Update READMEs (#36)
Genez-io/genezio-examples: javascript/chatgpt-project/README.md

[![Join our community](https://img.shields.io/discord/1024296197575422022?style=social&label=Join%20our%20community%20&logo=discord&labelColor=6A7EC2)](https://discord.gg/uc9H5YKjXv)
[![Follow @geneziodev](https://img.shields.io/twitter/url/https/twitter.com/geneziodev.svg?style=social&label=Follow%20%40geneziodev)](https://twitter.com/geneziodev)

</div>

# Rephrasing Using ChatGPT, Genezio, and ReactJS
In this project, we are creating an application that uses the ChatGPT API. It's backed by Genezio on the server side and features a React.js frontend.

## Demo - this can be optional

A hosted demo of this examples is available at [rephrasing-chatgpt.app.genez.io](https://rephrasing-chatgpt.app.genez.io).

## Prerequisites
- ✅ [NodeJs]() >= 16.0.0
- ✅ [npm]()
- ✅ [genezio]()

Note: We recommend using [nvm]() to install Node.js and npm.

## Project Structure

Inside the project folder, you will find the following files and folders:

```
├── backend/
│   ├── index.ts
│   └── models/
├── frontend/
│   ├── components/
│   ├── content/
│   ├── layouts/
│   └── pages/
├── genezio.yaml
├── README.md
├── .genezioignore
└── tsconfig.json

> ```

Reviewer: This is not accurate. The real file tree structure is different.

> Genezio looks for `genezio.yaml` to read the settings for deploying the project or for spinning a local dev server for testing.
> 
> The `backend` directory contains the implementation of the server side of the project.
> 
> The `frontend` directory contains a React application that talks with the genezio server.

Reviewer: ```suggestion
`genezio` looks for `genezio.yaml` to read the settings for deploying the project or for spinning a local dev server for testing.

The `backend` directory contains methods to communicate with the OpenAI API.

The `frontend` directory contains the source code for the React application.

To glue this two component together, an auto-generated SDK is installed in the `client/node_modules` folder.
This can be used by simply importing it into the frontend source code like any other dependency of your project.
```

```

Open a new terminal, navigate to the following directory, and run npm start to launch the React application:
```
cd ./client
npm run install-local-sdk
npm start
```

### Deploy your project with genezio

If you wish to deploy your project to the Genezio infrastructure, follow these steps:

Log in to Genezio using the command genezio login:
```
genezio login
```

Deploy your project using the genezio deploy command from the ``./genezio-examples/javascript/chatgpt-project`` directory.
```
genezio deploy
```

## Commands

All commands are run from the root of the project, from a terminal:

| Command                   | Action                                           |
| :------------------------ | :----------------------------------------------- |
| `npm install -g genezio`  | Installs genezio globally                        |
| `genezio login`           | Logs in to genezio                               |
| `genezio local`           | Starts a local server                            |
| `genezio deploy`          | Deploys a production project                     |
| `genezio --help`          | Get help using genezio                           |

## Want to learn more?

> Check out:
> - [ ] https://genez.io/docs
> - [ ] https://genez.io/blog
> - [ ] Discord channel

Reviewer: ```suggestion
Check out:
- [Official genezio documentation](https://genez.io/docs)
- [Web development tutorials](https://genez.io/blog)
- [Discord channel](https://discord.gg/uc9H5YKjXv)
```

## Contact

> If you need support or you have any questions, please join us in our [Discord channel](). We'd love to chat!

Reviewer: ```suggestion
If you need support or you have any questions, please join us in our [Discord channel](https://discord.gg/uc9H5YKjXv). We'd love to chat!
```
